packages/adp-tooling/src/types.ts

```diff
@@ -340,3 +364,59 @@ export interface AdpProjectData {
     reference: string;
     id: string;
 }
+
+export interface AdpCustomConfig {
+    adp: {
+        safeMode: boolean;
+        environment: OperationsType;
+    };
+}
+
+export interface InboundChangeContent {
+    inboundId: string;
+    entityPropertyChange: InboundChangeEntityPropertyChange[];
+}
+export interface InboundChangeContentAddInboundId {
+    inbound: {
+        [inboundId: string]: AddInboundModel;
+    };
+}
+export interface AddInboundModel {
+    semanticObject: string;
+    action: string;
+    title: string;
+    subTitle?: string;
+    signature: AddInboundSignitureModel;
+}
+export interface AddInboundSignitureModel {
+    parameters: InboundParameters;
+    additionalParameters: string;
+}
+export interface InboundParameters {
+    "sap-appvar-id"?: object;
+    "sap-priority"?: object;
+}
+export interface InboundChangeEntityPropertyChange {
+    propertyPath: string;
+    operation: string;
+    propertyValue: any;
+}
+export interface InboundChange {
+    inbound: {
+        [key: string]: {
+            semanticObject: string;
+            action: string;
+            icon: string;
+            title: string;
+            subTitle: string;
+            signature: {
+                parameters: object | string;
+                additionalParameters: "allowed";
+            };
+        };
+    };
+}
+
+export const enum FlpConfigurationType {
```

Comment:
What is this enum of one value?

Reply:
Currently, we are supporting only ADD_NEW_TILE, but expecting to be more in the future

Reply:
Could you give me a (few) examples? And will these also be called during the project generation?

Reply:
As far as I know, there is expected to be a "CHANGE TILE" as well

Reply:
Removed